src/components/Admin.js

```diff
@@ -9,17 +9,15 @@ class Admin extends React.Component {
   }
 
   loadAllNiceties = (callback) => {
-    $.ajax({
```

Comment:
Glad to be rid of this call to jQuery! I think this was the last reference to `$` in this file, so you can remove the import:

```
  2:8  warning  '$' is defined but never used  no-unused-vars
```